### DISCUSSION TITLE
How do I replace the starship binary with a version that has my local changes?

### DISCUSSION THREAD

**Discussion by rbren** [starship/starship#1633] [Q&A] [answered]
Didn't see anything in CONTRIBUTING.md

**Comment by matchai** [comment, marked as answer]
The following command should replace the installed version of Starship with the one with your local changes:
```sh
# In the starship project root
cargo install --path=. --force
```

**Comment by chipbuster** [comment]
Alternative options (if you only want to temporarily test something out) are to set your path so that the binary you want is located first in PATH (this is what I do), or to feed the absolute path to the starship binary as your init command.

Solution 1:

```
export PATH=/path/to/my/custom/binary:$PATH
starship init myshell   # Replace with the actual init command
```

Solution 2:
```
/path/to/my/custom/binary/starship init myshell
```
